Makefile

```diff
@@ -173,4 +173,9 @@ $(DEPS_BUILD_DIR)/%: phony_force
 	$(BUILD_TOOL) -C $(DEPS_BUILD_DIR) $(patsubst $(DEPS_BUILD_DIR)/%,%,$@)
 endif
 
+.PHONY: doc
+doc: api-metadata
```

Comment:
Specify dependencies with cmake instead of make. Use make only for the top level command as syntactic sugar.

Reply:
help?

Reply:
Ok, I think I've figured it out and made it good.